### PR TITLE
Address design feedback on Profile share tab Screen

### LINKED ISSUE #16080: Profile - Share Tab - Design Review
Following discussion with design team the current implementation of Profile Share tab requires some adjustments.

@mariocnovoa will do this review in Figma. Align with him and adjust as required.

### PR DESCRIPTION
**Pull request by siddarthkay**
fixes #16080

### Testing notes
No functionality was changed, only UI Touchups

#### Platforms
- Android
- iOS

status: ready

### CODE REVIEW

File: src/status_im2/contexts/share/view.cljs [hunk 1]
```diff
@@ -15,20 +15,30 @@
...
+     :i/close]
+    [quo/button
+     {:icon                true
+      :type                :blur-bg
+      :size                32
+      :accessibility-label :shell-scan-button
+      :override-theme      :dark
+      :on-press            #(rf/dispatch [:navigate-back])}
+     :i/scan]
+   ]
```
Comment (flexsurfer) [inline src/status_im2/contexts/share/view.cljs]: formatting

File: src/status_im2/contexts/share/view.cljs [hunk 2]
```diff
@@ -15,20 +15,30 @@
...
    [quo/text
     {:size   :heading-1
      :weight :semi-bold
      :style  style/header-heading}
-    (i18n/label :t/share)]])
+    (i18n/label :t/share)]
+  ])
```
Comment (flexsurfer) [inline src/status_im2/contexts/share/view.cljs]: formatting

File: src/status_im2/contexts/share/style.cljs [hunk 3]
```diff
@@ -5,32 +5,41 @@
...
 
 (def blur
-  {:style         {:position :absolute
-                   :top      0
-                   :left     0
-                   :right    0
-                   :bottom   0}
-   :overlay-color colors/neutral-80-opa-80
-   :blur-amount   20})
+  {:style            {:position :absolute
```
Comment (flexsurfer) [inline src/status_im2/contexts/share/style.cljs]: properties shouldn't be mixed in styles, see guidlines

File: src/status_im2/contexts/share/view.cljs (hunk 1 above)
Comment (siddarthkay) [reply src/status_im2/contexts/share/view.cljs]: ouch

Comment (siddarthkay) [reply src/status_im2/contexts/share/view.cljs]: resolved : https://github.com/status-im/status-mobile/pull/16257/commits/7d9cf0a802dcaa530cee38159fabd0ca33b29306

File: src/status_im2/contexts/share/view.cljs (hunk 2 above)
Comment (siddarthkay) [reply src/status_im2/contexts/share/view.cljs]: resolved : https://github.com/status-im/status-mobile/pull/16257/commits/7d9cf0a802dcaa530cee38159fabd0ca33b29306

File: src/status_im2/contexts/share/style.cljs (hunk 3 above)
Comment (siddarthkay) [reply src/status_im2/contexts/share/style.cljs]: resolved : https://github.com/status-im/status-mobile/pull/16257/commits/7d9cf0a802dcaa530cee38159fabd0ca33b29306

**Review by flexsurfer — APPROVED** [review]

**Comment by siddarthkay** [comment]
I Will wait to hear from @mariocnovoa on the design review as well before merging this in.

**Comment by J-Son89** [comment]
Nice, I'm also planning to check with figma this evening 👌
You done a review with figma + iPhone 11 Pro @siddarthkay?

**Review by ilmotta — APPROVED** [review]
LGTM

File: src/status_im2/contexts/share/view.cljs [hunk 4]
```diff
@@ -15,15 +15,23 @@
...
+      :on-press            #(rf/dispatch [:navigate-back])}
+     :i/close]
+    [quo/button
+     {:icon                true
+      :type                :blur-bg
+      :size                32
+      :accessibility-label :shell-scan-button
+      :override-theme      :dark
+      :on-press            #(rf/dispatch [:navigate-back])}
+     :i/scan]]
```
Comment (ibrkhalil) [inline src/status_im2/contexts/share/view.cljs]: Nitpick: The two buttons are identical and can be refactored into a function with three arguments, accessibility label, label, and content.

**Review by ibrkhalil — APPROVED** [review]
Non blocking comment

**Comment by siddarthkay** [comment]
> done a review with figma + iPhone 11 Pro @siddarthkay?

7pu6nj

File: src/status_im2/contexts/share/view.cljs (hunk 4 above)
Comment (siddarthkay) [reply src/status_im2/contexts/share/view.cljs]: Hi @ibrkhalil : Thanks for the review!
I love to abstract common logic and make re-usable functions, Although in this case it seems to be an overkill to make the suggested change.

**Comment by pavloburykh** [comment]
Hey @siddarthkay ! Is PR ready for test?

**Comment by siddarthkay** [comment]
Hi @pavloburykh : Sorry not yet, I've just now requested a re-review by Design Team. 
Once I get the green signal from them I'll ping QA for testing :)

**Comment by du82** [comment]
The background color seems a bit off compared to what I remember it being. Are you sure its not supposed to be blue? The figma designs are all locked down sadly I cant see the official design

**Comment by churik** [comment]
I believe should fix also https://github.com/status-im/status-mobile/issues/16874

**Comment by flexsurfer** [comment]
hey @siddarthkay still relevant? looks stale

File: src/status_im2/contexts/share/view.cljs (hunk 4 above)
Comment (ibrkhalil) [reply src/status_im2/contexts/share/view.cljs]: Understood, Thanks Sid!

**Comment by siddarthkay** [comment]
> hey @siddarthkay still relevant? looks stale

hmm yes will rebase and continue

**Comment by flexsurfer** [comment]
@siddarthkay could you please fix lint

**Comment by pavloburykh** [comment]
**QA notes:** the above test [run results](https://github.com/status-im/status-mobile/pull/16257#issuecomment-1691422449) are irrelevant as testrun has been started on outdated build. 

Rerun is needed after PR will be rebased.

**Comment by qoqobolo** [comment]
Hey @siddarthkay, thanks for the fixes!
There are still some issues 

### ISSUE 1: No close ( `x` ) button

Design:
Screenshot 2023-08-29 at 17 10 29
 
PR:
Screenshot 2023-08-29 at 17 10 58

### ISSUE 2: Still no `Scan` button

Design:

Screenshot 2023-08-29 at 17 20 15


### ISSUE 3: Blur overlay color

Not sure about this one, but it still looks different compared to the design (still looks grey, especially on Android, and too dark on iOS), could you recheck, please?

Screenshot 2023-08-29 at 17 14 25

Design:
Screenshot 2023-08-29 at 17 19 06


PR (Android and iOS):

Screenshot 2023-08-29 at 17 16 44
Screenshot 2023-08-29 at 17 16 57

### ISSUE 4 (minor): Link to profile overlaps the `Share` icon on small screen

Can this be adjusted? 

Screenshot 2023-08-29 at 17 17 54

**Comment by siddarthkay** [comment]
@qoqobolo @pavloburykh @churik : I've fixed all the issues.
note on issue 4: I've made some updates to make blur better (blur not matching 100% with designs is a known issue)
This PR is ready for one more round of testing.

Apologies for the delay.

**Comment by qoqobolo** [comment]
@siddarthkay thanks for the fixes!
One question.

I see you have updated the link format on the `Share` screen, but I'm afraid because of this, adding contacts by scanning a QR code doesn't work anymore. 
In order for this to work correctly, we need to carry out full work on updating the format https://github.com/status-im/status-mobile/issues/16810
Please correct me if I misunderstood the purpose of this change.

Screenshot 2023-09-15 at 11 30 34


https://github.com/status-im/status-mobile/assets/67952253/8044d7cf-8980-4960-844d-17dacdaa88b6

**Comment by siddarthkay** [comment]
Hi @qoqobolo : you're right, I missed this, I'll revert the links back to old format.

**Comment by qoqobolo** [comment]
Thanks for your work @siddarthkay!
PR is tested and ready to be reviewed by @Francesca-G

**Review by Francesca-G — APPROVED** [review]
Looks good to me ✨

**Pull request merged**